src/agl/web_app_manager_service_agl.cc

```diff
@@ -412,10 +412,8 @@ void WebAppManagerServiceAGL::LaunchStartupAppFromConfig() {
       [&](const auto& extension) { extensions_obj.append(extension); });
   obj["extensions"] = extensions_obj;
 
-  if (width)
-    width_ = atoi((const char*)width);
-  if (height)
-    height_ = atoi((const char*)height);
+  width_ = width ? atoi((const char*)width) : 0;
```

Comment:
Wouldn't be clearer to just to init the values to 0?

Reply:
Values are initialized to 0 already. Problem is that service is shared among all the launch attempts, and there was no code resetting the values between attempts. Thus, we explicitely say in this code "if there is no value in the incoming launch flags, set to 0".

Reply:
Yeah, I initial commented on this, and then saw the commit description, and added that comment. Fair enough.